testsuite/MDAnalysisTests/transformations/test_fit.py

```diff
@@ -99,8 +99,13 @@ def test_fit_translation_no_masses(fit_universe):
     # create a universe without masses
     ref_u  = make_Universe()
     # what happens Universe without masses is given?
-    with pytest.raises(AttributeError): 
```

Comment:
This will not work since your code will always raise an error. Revert the changes and only change the 
`AttributeError` to an `TypeError`

I would also recommend that you test your changes locally with pytest. How to test your code is explained in the [wiki](https://github.com/MDAnalysis/mdanalysis/wiki/UnitTests).

Reply:
Also the [user guide](https://www.mdanalysis.org/UserGuide/testing.html#running-the-test-suite).